pyogrio/core.py

```diff
@@ -1,27 +1,38 @@
+"""Core functions to interact with OGR data sources."""
+
 from pyogrio._env import GDALEnv
 from pyogrio.util import (
-    get_vsi_path_or_buffer,
-    _preprocess_options_key_value,
     _mask_to_wkb,
+    _preprocess_options_key_value,
+    get_vsi_path_or_buffer,
 )
 
-
 with GDALEnv():
+    from pyogrio._err import _register_error_handler
+    from pyogrio._io import ogr_list_layers, ogr_read_bounds, ogr_read_info
     from pyogrio._ogr import (
+        _get_drivers_for_path,
+        _register_drivers,
+        get_gdal_geos_version,
         get_gdal_version,
         get_gdal_version_string,
-        get_gdal_geos_version,
         ogr_list_drivers,
-        set_gdal_config_options as _set_gdal_config_options,
+    )
```

Comment:
any idea why these got split up into multiple blocks?  Seems like all imports from `pyogrio._ogr` should be in a single block.

Reply:
The default behaviour is apparently to keep import with an alias on a seperate line rather than combining them. This can be changed with an extra configuration setting: `combine_as_imports = true`.

I'll add it in the config...